src/GUI/ReverseEngineer20/ReverseEngineer/InflectorPluralizer.cs

```diff
@@ -1,14 +1,36 @@
 ﻿using Humanizer;
 using Microsoft.EntityFrameworkCore.Design;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace ReverseEngineer20.ReverseEngineer
 {
     public class InflectorPluralizer : IPluralizer
     {
+        private readonly Dictionary<string, string> _customerCharacterPluralize;
+        private readonly Dictionary<string, string> _customerCharacterSingularize;
+
+        public InflectorPluralizer(List<Schema> customReplacers)
+        {
+            _customerCharacterPluralize = customReplacers
+                .SelectMany(x => x.Tables)
+                .Where(x => !string.IsNullOrWhiteSpace(x.VariableName))
+                .ToDictionary(key => key.Name.ToPascalCase(), value => value.VariableName);
+
+            _customerCharacterSingularize = customReplacers
```

Comment:
Will this not always block standard renaming from working?

Reply:
I don't known what's your meaning
I think If you want to custom name, you only want your name is equals your naming not be transfer by other function?